code/controllers/subsystem/gravity.dm

```diff
@@ -60,6 +63,7 @@ var/global/legacy_gravity = FALSE
 	recalculation_cost = REALTIMEOFDAY - tempcost
 
 /datum/subsystem/gravity/fire(resumed = FALSE)
+	set waitfor = 0
```

Comment:
Don't do this here, the mc has a system for detecting sleeps in subsystems that this would break. there is nothing worst then two fire's running at the same time.

This should be done in gravity_act and/or sync_gravity or what ever child proc you call that sleeps